docs/src/content/docs/challenges/forms/40-control-value-accessor.md

```diff
@@ -0,0 +1,20 @@
+---
+title: 🟠 Control Value Accessor
+description: Challenge 40 is about creating a custom form control that implements Control Value Accessor interface.
+author: stanislav-gavrilov
+challengeNumber: 40
+command: forms-control-value-accessor
+sidebar:
+  order: 119
+  badge: New
+---
+
+:::note
+WIP: The following documentation will be reviewed and improved. However, you can still take on the challenge. If you don’t understand a certain part, please feel free to reach out or create an issue.
+:::
```

Comment:
You can delete this comment, the documentation will be readable when we merge it. 😜

Reply:
deleted